Tests/UnitTests/Purchasing/StoreKitAbstractions/SubscriptionPeriodTests.swift

```diff
@@ -141,7 +141,7 @@ class SubscriptionPeriodTests: TestCase {
 
         let expectations: [(
             inputValue: Int, inputUnit: SKProduct.PeriodUnit,
-            expectedValue: Int, expectedUnit: SubscriptionPeriod.Unit
+            expectedValue: Int, expectedUnit: RevenueCat.SubscriptionPeriod.Unit
```

Comment:
This fixes a test build error about conflicting type with `StoreKit`s `SubscriptionPeriod`